src/main/resources/META-INF/plugin-file-view-provider-ce.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This is an optional dependency `plugin.xml` file that's only loaded if the plugin is being used in IntelliJ IDEA Community Edition or PyCharm Community Edition. In those IDEs, `CSS` and `JavaScript` are abstract file types whereas in all commercial JetBrains IDEs, they are first-class language file types. As a result, we can safely register the semantic tokens-based file view provider factory for those file types in the free Community Edition IDEs.